hubclient/client.go

```diff
@@ -437,6 +443,11 @@ func (c *Client) applyHeaderValues(request *http.Request) {
 		}
 
 	}
+
+	if c.userAgent != "" {
+		request.Header.Add("User-Agent", c.userAgent)
```

Comment:
`User-Agent` we could move this with other constants here: https://github.com/blackducksoftware/hub-client-go/blob/master/hubclient/constants.go

Reply:
Done